spec/integrations/live_testing_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'Live testing' do
+  before do
+    skip('No live testing') unless ENV['LIVE']
+  end
+
+  context 'with a real UDP socket' do
+    it 'should actually send stuff over the socket' do
+      socket = UDPSocket.new
+      host, port = 'localhost', 12345
+      socket.bind(host, port)
+
+      statsd = Datadog::Statsd.new(host, port)
+      statsd.increment('foobar')
+      message = socket.recvfrom(64).first
+      expect(message).to eq 'foobar:1|c'
```

Comment:
Should we test more than just a counter? Probably not has you improved the test coverage on everything but if that's like 2, 4 more lines, could be helpful someday.

Reply:
We could make a big test for that. I'll add it.